content/docs/01-introduction/6-free-cloud-credit

```diff
@@ -0,0 +1,29 @@
+---
+title: "Free Cloud Credit?"
+metaTitle: "About Free Cloud Credit"
+metaDescription: "Concepts of Free Cloud Credit"
+icon: ""
+hideToC: true
+fullWidth: false
+---
+
+# Free Cloud Credit with Palette
+
+Free Cloud Credit is offered to customers who do not have access to a cloud account and want to try Palette, before committing to the Palette platform. Perhaps a customer finds themselves without sufficient permissions to input their existing cloud credentials or is exploring new ways to manage their services. The Free Cloud Credit, granted through the Palette account, is a great way to begin exploring the Palette platform. 
+
+
+## Requesting a Free Cloud Account
+
+To request an authorization to the Spectro Cloud Free Cloud Credit program, connect with Gilbert Tang, via the Slack Spectro Cloud Community and ask about the Free Cloud Credit program. A $100 of free cloud credit will be granted to use with the Spectro Cloud's Palette platform. The user is encouraged to monitor the usage and expenditure percentage through the Palette and as they reach the conclusion of the credit usage, a message will remind the user of the Usage Status.
+
+
+## Managing a Free Cloud Account
+
+The Free Cloud Credit ends when the $100 of the free cloud credit granted by Spectro Cloud is consumed. At that time, all resources created during the trial period will cease services and clusters will be marked for deletion and will be lost, so make sure to back up as needed. A message stating that the Free Cloud Credit has been suspended will be sent to inform the user at the 50% consumption stage.  
+
+
+## Upgrading to a Paid Cloud Account
+
+You can upgrade to a Cloud Account at any time after trying the Free Cloud Credit service. Create your own Cloud Account and replace the Free Cloud Credit with your own Cloud Account information.
+
+
```

Comment:
3 mail Notifications: [**Plz check on this info if correct we can include**]
-------------------------------------------
**Usage:     Notification Send to:**
-------------------------------------------
 >50%      Tenant
>90%       Tenant
>100%     Tenant and system admin


Tenant will have access to: [**Plz check on this info if correct we can include**]
-------------------------------
**AWS account
Iamusername, password**
**access key and secret key** for resource creation purposes

Reply:
or lets not be specific to any cloud we can just go with:

Palette provides cloud account- with required credentials-  for user deployments.